yb_migrate/src/srcdb/utils.go

```diff
@@ -0,0 +1,12 @@
+package srcdb
+
+import (
+	"fmt"
+	"log"
+	"os"
+)
+
+func ErrExit(formatString string, args ...interface{}) {
```

Comment:
is there some issue in using this function from utils package directly?

Reply:
As of now, `utils` is importing `srcdb` for the `Source` type. Importing `utils` in the `srcdb` package will create a circular import.

If I manage to remove `srcdb` imports from the `utils` package, I will consider using `utils` in this package.

Reply:
got it